src/ol_infrastructure/applications/mit_learn/Pulumi.applications.mit_learn.Production.yaml

```diff
@@ -75,6 +75,8 @@ config:
     OPENSEARCH_URL: "https://search-opensearch-mitlearn-producti-qjtnw2dcoampmbtscs3wdfci6y.us-east-1.es.amazonaws.com"
     POSTHOG_API_HOST: "https://ph.ol.mit.edu"
     POSTHOG_ENABLED: "true"
+    POSTHOG_EVENT_S3_BUCKET: "ol-data-lake-landing-zone-production"
+    POSTHOG_EVENT_S3_PREFIX: "thirdparty/posthog/learn/"
```

Comment:
This also has `events` in the prefix, to differentiate from `persons` which I also set an export for to ingest into the data platform.